CHANGELOG.md

```diff
@@ -17,6 +17,10 @@ Changes are grouped as follows
 - `Fixed` for any bug fixes.
 - `Security` in case of vulnerabilities.
 
+## [6.28.6] - 2023-10-04
+### Added
+- Convenience method `to_text` on the `FunctionCallLog` class that formats log entries into a new-line delimited string.
```

Comment:
I think a better formulation would be to state how this helps the user nicely printing the output logs of a function call 😄